x/accounts/defaults/lockup/TUTORIAL.md

```diff
@@ -0,0 +1,246 @@
+# Using lockup account on Cosmos sdk
+
+* [Setup](#setup)
+* [Init](#init)
+* [Execution](#execution)
+    * [Delegate](#delegate)
+    * [Undelegate](#undelegate)
+    * [Withdraw reward](#withdraw-reward)
+    * [Withdraw unlocked token](#withdraw-unlocked-token)
+    * [Send coins](#send-coins)
+* [Query](#query)
+    * [Query account info](#query-account-info)
+    * [Query periodic lockup account locking periods](#query-periodic-lockup-account-locking-periods)
+
```

Comment:
Let's add an abstract/summary to link the readme and have a quick refresher on what this is.